x-pack/plugin/build.gradle

```diff
@@ -113,6 +113,32 @@ tasks.named("transformV7RestTests").configure({ task ->
   task.replaceKeyInDo("rollup.stop_job", "xpack-rollup.stop_job")
   task.addAllowedWarningRegex(".*_xpack/rollup.* is deprecated.*")
 })
+def v7compatiblityNotSupportedTests = {
+  return [
+    // to support it, it would require to almost revert back the #48725 and complicate the code
+    'vectors/10_dense_vector_basic/Deprecated function signature',
+    // not going to be supported
+    'vectors/30_sparse_vector_basic/Cosine Similarity',
+    'vectors/30_sparse_vector_basic/Deprecated function signature',
+    'vectors/30_sparse_vector_basic/Dot Product',
+    'vectors/35_sparse_vector_l1l2/L1 norm',
+    'vectors/35_sparse_vector_l1l2/L2 norm',
+    'vectors/40_sparse_vector_special_cases/Dimensions can be sorted differently',
+    'vectors/40_sparse_vector_special_cases/Documents missing a vector field',
+    'vectors/40_sparse_vector_special_cases/Query vector has different dimensions from documents\' vectors',
+    'vectors/40_sparse_vector_special_cases/Sparse vectors should error with dense vector functions',
+    'vectors/40_sparse_vector_special_cases/Vectors of different dimensions and data types',
+    // the test uses sparse vector - not supported
+    'vectors/50_vector_stats/Usage stats on vector fields',
+
+    // ML - not needing compatible api
+    // some are failing due to cat api being called with JSON accept header
+    'ml*/*/*',
```

Comment:
Can't the list of ML tests that don't work just stay as it was before instead of being expanded to include all ML tests?  If not then it must be that this PR has introduced new problems for ML.

Even though the list of ML tests before was quite big, it was only a small fraction of all ML tests.  Adding a total wildcard will mean others that should work are accidentally broken between now and 8.0 release.

Reply:
good suggestion. I will remove the wildcard. I mistakenly assumed that all ML endpoints are not available to end users - which obviously not true.
I will remove the wildcard and see which exactly fail now